.gitignore

```diff
@@ -4,6 +4,7 @@
 /node_modules
 /.pnp
 .pnp.js
+package-lock.json
```

Comment:
Why this?

Reply:
Koi sense hai remote pe rakhne ka ise?

Reply:
Time km lgega naa dependencies install krne ka fir

Reply:
Fixed